docs/custom-images.md

```diff
@@ -0,0 +1,165 @@
+    sed -i -e "s/^IMAGE=.*/IMAGE=\"${IMAGE_REPO}:${IMAGE_TAG}\"/g" /etc/os-release
+
+# IMPORTANT: it is good practice to recreate the initrd and re-apply `elemental-init`
+# command that was used in the base image. This ensures that any eventual change that should
+# be synced in initrd included binaries is also applied there and consistent.
+RUN elemental init --force immutable-rootfs,grub-config,dracut-config,cloud-config-essentials,elemental-setup
+```
+
+Where `latest` is the base version we want to customize.
+
+And then the following commands
+
+```bash showLineNumbers
+docker build --build-arg IMAGE_REPO=myrepo/custom-build \
+             --build-arg IMAGE_TAG=v1.1.1 \
+             -t myrepo/custom-build:v1.1.1 .
+docker push myrepo/custom-build:v1.1.1
+```
+
+The new customized OS is available as the Docker image `myrepo/custom-build:v1.1.1` and it can
+be run and verified using docker with
+
+```bash showLineNumbers
+docker run -it myrepo/custom-build:v1.1.1 bash
+```
+
+## Create a custom bootable installation ISO
+
+Elemental leverages container images to build its root filesystems; therefore, it is possible
+to use it in a multi-stage environment to create custom bootable media that bundles a custom container image.
+
+```docker showLineNumbers
+FROM registry.suse.com/suse/sle-micro/5.5:latest AS os
+
+# Check the previous section on building custom images
+
+# The released OS already includes the toolchain for building ISOs
+FROM registry.suse.com/suse/sle-micro/5.5:latest AS builder
+
+ARG TARGETARCH
+WORKDIR /iso
+COPY --from=os / rootfs
+
+# work around buildah issue: https://github.com/containers/buildah/issues/4242
+RUN rm -f rootfs/etc/resolv.conf
+
+RUN elemental build-iso \
+        dir:rootfs \
+        --bootloader-in-rootfs \
+        --squash-no-compression \
+        -o /output -n "elemental-${TARGETARCH}"
+
+FROM busybox
+COPY --from=builder /output /custom-iso
+
+ENTRYPOINT ["busybox", "sh", "-c"]
+```
+
+Build it with regular `docker build` command:
+
+```bash showLineNumbers
+docker build -t myrepo/custom-build:v1.1.1 \
+              --build-arg IMAGE_REPO=myrepo/custom-build-iso \
+              --build-arg IMAGE_TAG=v1.1.1 \
+              .
+```
+
+The resulting container image is actually a container image including the ISO,
+this container image can be pushed to an OCI registry too. The ISO image can be
+extracted from the container to the current folder by executing the container as:
+
+```bash showLineNumbers
+docker run --rm -v $(pwd):/host mytest-image "busybox cp /custom-iso/*.iso /host"
+```
+
+The new customized installation media can be found in `elemental-<arch>.iso`.
+
+The above container run is equivalent to what *elemental-operator* does to extract
+the ISO from a container to build a new one including the registration URL,
+hence this is also a good check mark to verify the container can be pushed to a
+registry and used in *elemental-operator* as a `baseImage` for a
```

Comment:
I see why you used "in" here... anyway, seems to me sounds better (and may be more clear to the casual user) if we use "by" here...
btw, drop the suggestion if you prefer "in".
```suggestion
registry and used by the *elemental-operator* as a `baseImage` for a
```